pds_github_util/issues/move_issues.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+import logging
+import github3
+from datetime import datetime
+
+
+# Quiet github3 logging
+logger = logging.getLogger('github3')
+logger.setLevel(level=logging.WARNING)
+
+# Enable logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def create_parser():
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     description=__doc__)
+    parser.add_argument('--source-repo', type=str, required=True,
+                        help='source repository <org>/<repo>')
+    parser.add_argument('--target-repo', type=str,
+                        help='target repository <org>/<repo>')
+    parser.add_argument('--label', type=str, required=False, default=None,
+                        help='label added to the issues in their target repository, '
+                             'usually the name of the module the issue is related to')
+    parser.add_argument('--token', type=str, required=False,
+                        help='github token.')
+    parser.add_argument('--dry-run', default=False, required=False, action='store_true')
+    return parser
+
+
+def get_gh_connection(token=None):
+    github_token = token or os.environ.get('GITHUB_TOKEN')
+    if not github_token:
+        logger.error(f'github API token must be provided or set as environment'
+                     ' variable (GITHUB_TOKEN).')
+        sys.exit(1)
+    else:
+        return github3.login(token=github_token)
+
+
+def move_issue(issue, target_repository, label=None, dry_run=False):
+    labels = [label.name for label in issue.labels()]
+    if label:
+        labels.append(label)
+
+    comment = {
+        'created_at': datetime.now().astimezone().isoformat(),
+        'body': f'imported, see original ticket {issue.html_url}'
+    }
+
+    optional_args = {
+        'closed': issue.state == 'closed',
+        'labels': labels,
+        'comments': [comment]
+    }
+    if issue.assignee:
+        optional_args['assignee'] = issue.assignee.login
+    if issue.milestone:
+        optional_args['milestone'] = issue.milestone.id
+
+    if not dry_run:
+        target_repository.import_issue(
+            issue.title,
+            issue.body,
+            issue.created_at.isoformat(),
+            **optional_args)
+
+    if issue.state != 'closed':
+        issue.close()
+
+
+def move_issues(source_repo, target_repo, gh_connection, label=None, dry_run=True):
+    [owner, repo_name] = source_repo.split("/")
```

Comment:
Don't need an intermediate temporary `list` object here since tuple-unpacking happens automatically:
```python
owner, repo_name = source_repo.split('/')
```